projects/ui/src/components/Common/Charts/ChartPropProvider.tsx

```diff
@@ -135,6 +135,7 @@ export type BaseChartProps = {
   curve?: CurveFactory | keyof typeof CURVES;
   scale?: keyof typeof SCALES;
   isTWAP?: boolean;
+  isZeroHorizontal?: boolean;
```

Comment:
@k-xo Can you walk me thru why you chose this route instead condensing both into one var like `horizontalLine: number`?

Reply:
Ah yes, was getting the zeroHorizontal approach to work, but as mentioned final version would be better with a generalised version, will make this change now